.spi.yml

```diff
@@ -0,0 +1,19 @@
+# This is manifest file for the Swift Package Index for it to 
+# auto-generate and host DocC documentation.
+# For reference see https://blog.swiftpackageindex.com/posts/auto-generating-auto-hosting-and-auto-updating-docc-documentation/.
+
+version: 1
+builder:
+  configs:
+    - documentation_targets: 
```

Comment:
Yes, it is!

Just in case you're doing more of these and want to double check, you can use "Try in a Playground" with the underlying package which we use to parse the file, [SPIManifest](https://swiftpackageindex.com/SwiftPackageIndex/SPIManifest), to check the format.

We have plans to make this a linter page on SPI at some point to remove the friction around this :)

<img width="1462" alt="CleanShot 2022-12-04 at 18 45 13@2x" src="https://user-images.githubusercontent.com/65520/205506714-5ed85f86-3a23-40c1-b6f0-fbbb2f8d4176.png">

Reply:
Or just tag me in and I'll check them, no problem!

Reply:
I assume there's no way to tell Swift Package Index that it should build documentation for all targets, right?

Reply:
Unfortunately there isn't at this time.

Reply:
Thanks @finestructure!